build/main.go

```diff
@@ -169,15 +171,62 @@ func forceBuildFC() {
 	}...))
 }
 
+func cleanDirectory(dir string, ignoredots bool) error {
```

Comment:
Can you add a small comment stating that we use this method (and not https://golang.org/pkg/os/#RemoveAll) to preserve the `.gitkeep` file? (or could rename the method to reflect that).

Is the `.gitkeep` necessary?

Reply:
Ya, the `.gitkeep` is necessary if we want the directory to exist on code checkout.